File: aAlvarezAssignment4/ElectricCar.java
package aAlvarezAssignment4;

public class ElectricCar extends Car{
    private double batterySize;

    ElectricCar(String makeAndModel, int maximumNumberOfPassengers, int numberOfDoors, double batterySize){
        super(makeAndModel, maximumNumberOfPassengers, numberOfDoors);
        this.batterySize= batterySize;
}
    public void setBatterySize(double b){
        this.batterySize= b;
    }

    public double getBatterySize(){
        return batterySize;
    }

    public String toString(){
        return (super.toString() + "\nBattery Size: " + batterySize);
    }
}

File: aAlvarezAssignment3/ElectricCar.py
from Car import Car
class ElectricCar(Car):
    def __init__(self,makeAndModel="none assigned", numberOfDoors=4, maximumNumberOfPassengers=5, batterySize=-1):
        super().__init__(makeAndModel,numberOfDoors,maximumNumberOfPassengers)
        self.battery= batterySize
    
    def getBatterySize(self):
        return self.battery
    
    def setBatterySize(self,size):
        self.battery= size
    
    def __str__(self):
        return super().__str__() + "\nBattery Size: " + str(self.battery)
    
